server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -304,6 +312,34 @@ private static QueryBuilder toValueQuery(MetricCriterion criterion) {
     }
   }
 
+  public List<String> searchTags(@Nullable String textQuery, int pageSize) {
+    checkArgument(pageSize <= 100, "Page size must be lower than or equals to " + 100);
+    if (pageSize == 0) {
```

Comment:
Other WS on rules and issues tags do not have the same behavior => we should fix them (create JIRA tickets) or change this one